### PR TITLE
Fix agency filtering

### PR DESCRIPTION
**Pull request by hartsick**
An attribute (xml) we need for displaying the SORNS was accidentally
left out of the list of attributes the agency filtering scope returns.
To fix and avoid this in the future, we now select *all* database
attributes when using the scope.

Also updates a test to make sure code path it tests is triggered;
parameter was incorrectly named before, making it pass incorrectly.

Signed-off-by: Natasha Ibrahim <natasha.ibrahim@gsa.gov>

Closes https://github.com/18F/all_sorns/issues/195

### CODE REVIEW

**Comment by hartsick** [comment]
@NatashaJIbrahim I put in what I think is the most straightforward fix that hopefully will also help us in the future. Would you mind taking a look? I dug a little bit more into that scope to see if we could remove it and just use distinct alone to de-dupe, but something in later calls relies on the pg_search rank. I decided to just move forward with the small fix as a result, but let me know if you want to dig in!

If it looks good, let me know and I'll change this from a draft to ready for review so someone on the SORNS team can merge.

**Comment by hartsick** [comment]
One thing I just noticed that we should decide whether to address—the following currently returns an error in this PR and on main:
* Search 'elections', see results
* On lefthand side under 'agencies', search 'eac' and click the box for 'Election Assistance Commission'
* See results filter (yay!)
* On same screen, empty search box and click 'Search'
* See results stay same and 500 in console

**Comment by NatashaJIbrahim** [comment]
@hartsick This looks good to me. I think the rendering error you found could go either way and be opened as another ticket or be added her as a second commit. It's a little scope creep but if it seems quick and easy no reason to leave it hanging. I'll defer to you and someone on the SORNS team on that.

**Review by peterrowland — APPROVED** [review]
√

**Comment by folksgl** [comment]
@peterrowland was this deployed to `test` and validated before merging?

**Pull request merged**
